### PR TITLE
[MI-2638] Update slash commands

### PR DESCRIPTION
**Pull request by ayusht2810**

### CODE REVIEW

File: server/plugin/command.go [hunk 1]
```diff
@@ -236,8 +236,18 @@ func (p *Plugin) handleCreate(args *model.CommandArgs, parameters []string, clie
 	}
 }
 
-func (p *Plugin) handleSearchAndShare(args *model.CommandArgs, params []string, client Client, _ bool) string {
-	return ""
+func (p *Plugin) handleRecords(args *model.CommandArgs, parameters []string, client Client, isSysAdmin bool) string {
+	if len(parameters) == 0 {
+		return "Invalid create command. Available commands are 'share' and 'view'."
```
Comment (Kshitij-Katiyar) [inline server/plugin/command.go:241]: is this a create command ?

File: server/plugin/command_test.go [hunk 2]
```diff
@@ -775,31 +775,32 @@ func TestParseCommand(t *testing.T) {
...
-			expectedAction:     "subscriptions",
+			expectedAction:     constants.CommandSubscriptions,
 			expectedParameters: []string{constants.SubCommandDelete, "mockID"},
 		},
 		{
-			description:    "ParseCommand: share command",
-			input:          "/servicenow share",
-			expectedAction: constants.CommandSearchAndShare,
+			description:        "ParseCommand: records command",
+			input:              "/servicenow records share",
```
Comment (Kshitij-Katiyar) [inline server/plugin/command_test.go:801]: at a time i think you can share one record if that's the case I think the command should be  "/servicenow record share" 
if not keep it as it is

Comment (ayusht2810) [reply server/plugin/command_test.go:801]: in the subscription command as well we have kept it as `subscriptions`, so I think it is fine in this way.

Comment (Kshitij-Katiyar) [reply server/plugin/command_test.go:801]: ok sure but still confirm this once

**Review by Kshitij-Katiyar — APPROVED** [review]

File: server/plugin/command.go [hunk 3]
```diff
@@ -236,8 +236,18 @@ func (p *Plugin) handleCreate(args *model.CommandArgs, parameters []string, clie
 	}
 }
 
-func (p *Plugin) handleSearchAndShare(args *model.CommandArgs, params []string, client Client, _ bool) string {
-	return ""
+func (p *Plugin) handleRecords(args *model.CommandArgs, parameters []string, client Client, isSysAdmin bool) string {
```
Comment (manojmalik20) [inline server/plugin/command.go:239]:
```suggestion
func (p *Plugin) handleRecords(_ *model.CommandArgs, parameters []string, _ Client, _ bool) string {
```

File: server/plugin/command.go [hunk 4]
```diff
@@ -411,8 +421,11 @@ func getAutocompleteData() *model.AutocompleteData {
 
 	serviceNow.AddCommand(subscriptions)
 
-	searchRecords := model.NewAutocompleteData(constants.CommandSearchAndShare, "", "Search and share a ServiceNow record")
-	serviceNow.AddCommand(searchRecords)
+	records := model.NewAutocompleteData(constants.CommandRecords, "[command]", fmt.Sprintf("Available command: %s", constants.SubCommandSearchAndShare))
+
+	searchRecords := model.NewAutocompleteData(constants.SubCommandSearchAndShare, "", "Search and share a ServiceNow record")
+	records.AddCommand(searchRecords)
```
Comment (manojmalik20) [inline server/plugin/command.go:427]:
```suggestion
	shareRecords := model.NewAutocompleteData(constants.SubCommandSearchAndShare, "", "Search and share a ServiceNow record")
	records.AddCommand(shareRecords)
```

**Review by manojmalik20 — APPROVED** [review]

**Review by manojmalik20 — APPROVED** [review]

**Pull request merged**